prov/efa/src/rdm/efa_rdm_ep_fiops.c

```diff
@@ -941,6 +944,23 @@ static int efa_rdm_ep_close(struct fid *fid)
 		efa_rdm_ep->peer_srx_ep = NULL;
 	}
 
+	/* Remove all peers associated with this endpoint in domain level peer lists */
+	ofi_genlock_lock(&domain->srx_lock);
```

Comment:
Nit: Can u make L 948 and L927 use the same function to get lock ... I don't have preference on which one you use.

Reply:
Fixed, thanks